encoders/image/CustomKerasImageEncoder/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM jinaai/jina
+
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
+
+# install the third-party requirements
+RUN apt-get update && apt-get install --no-install-recommends -y git curl && \
```

Comment:
Are theses installs needed?

Reply:
these are not necessary here, removed them.